File: apps/api-gateway/src/main/java/com/seasonlink/config/UriConfig.java
package com.seasonlink.config;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties(prefix = "seasonlink.services")
public class UriConfig {
    private String jobs;
    private String jobOffers;
    private String companies;
    private String applications;
    private String profiles;
    private String ratings;
    private String messagery;
    private String notifications;
}

File: apps/gateway/src/main/kotlin/com/seasonlink/gateway/config/UriConfig.kt
package com.seasonlink.gateway.config

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * Configuration properties for the URIs of the services.
 * The URIs are configured in the application.yml file.
 */
@ConfigurationProperties(prefix = "seasonlink.services")
data class UriConfig(
    val jobs: String,
    val jobOffers: String,
    val companies: String,
    val profiles: String,
    val ratings: String,
    val notifications: String,
    val chat: String,
)
